Controllers/Article.controller.js

```diff
@@ -0,0 +1,13 @@
+const selectArticle = require("../Models/Article.model.js");
+
+const getArticle = (req, res) => {
+  selectArticle(req.params.article_id).then((article) => {
+    if(article){
+        res.status(200).send(article);
```

Comment:
Needs to be sent on the key of article

Reply:
Hi alex
im not sure if i get what you mean. Could you please clarify?

Reply:
Hi Hedieh. As it stands, you are sending the article object itself back as a response but it needs to be within another object with a key of article. You can do this by wrapping `article` on line 7 in curly brackets so it looks like `{ article }`